backend/corpora/lambdas/api/v1/curation/collections/common.py

```diff
@@ -151,6 +151,7 @@ class EntityColumns:
         "disease",
         "organism",
         "tombstone",
+        "donor_id",
```

Comment:
per brian and jason's [specifications](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell/308), we don't want the donor_id as part of the dataset preview (but we do want it as part of the full dataset metadata response)